README.md

```diff
@@ -31,8 +31,7 @@ order to:
 * Strip filenames and shuffle position information
 * Strip debugging information and symbol tables
 * Obfuscate literals, if the `-literals` flag is given
-* Expose [additional functions](#runtime-api) in the runtime package that can
-  optionally hide information during execution
+* Removes unnecessary data if the `-tiny` flag is given
```

Comment:
I'm not sure I would say unnecessary, because e.g. being able to un-garble panic messages with their line numbers could be necessary to some users.

Perhaps say "extra information" instead of "unnecessary data", and make those two words link to the "tiny mode" section.